cli/src/semgrep/core_runner.py

```diff
@@ -1087,14 +1098,10 @@ def _run_rules_direct_to_semgrep_core_helper(
                 print(" ".join(printed_cmd))
                 sys.exit(0)
 
-            runner = StreamingSemgrepCore(
-                # We expect to see 3 dots for each target, when running interfile analysis:
-                # - once when finishing phase 4, name resolution, on that target
-                # - once when finishing phase 5, taint configs, on that target
-                # - once when finishing analysis on that target as usual
-                cmd,
-                plan.num_targets * 3 if engine.is_interfile else plan.num_targets,
-            )
+            # Multiplied by three, because we have three places in Pro Engine to
+            # report progress, versus one for OSS Engine. This way, we don't get
+            # floating-point drift.
```

Comment:
```suggestion
            # Multiplied by three, because we have three places in Pro Engine to
            # report progress, versus one for OSS Engine.
```

I think this solution is logical enough that it doesn't need to be compared to incrementing by 1/3. In fact, I think bringing up floating-point drift out of nowhere (without introducing the alternative solution) is more confusing than not mentioning it.

Reply:
ooh, good point, i wasnt thinking about that